presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -81,6 +91,21 @@ public void copyFileToContainer(Transferable transferable, String containerPath)
         copyFileToContainer(containerPath, () -> super.copyFileToContainer(transferable, containerPath));
     }
 
+    public DockerContainer withExposedLogPaths(String... logPaths)
+    {
+        this.logPaths.addAll(Arrays.asList(logPaths));
+        return this;
+    }
+
+    public void exposeLogsInHostPath(Path hostBasePath)
+    {
+        for (String containerLogPath : logPaths) {
```

Comment:
there can be quite a lot of those. Is there a limit on number of directories bound...
I guess not very low - we used to bind a lot of stuff before moving to copying.

Reply:
Neither docker java nor docker docs itself mention any limit on the number of binds